### PR TITLE
Add xdg-open along with open and friendly error output on REPORT_OUTPUT failure

### PR DESCRIPTION
**Pull request by exapsy**

### CODE REVIEW

**Comment by alixander** [comment]
could you set up commit signing please? https://github.com/terrastruct/d2/pull/557#issuecomment-1367468730

File: ci/e2ereport.sh [hunk 1]
```diff
@@ -9,7 +9,9 @@ FORCE_COLOR=1 DEBUG=1 go run ./e2etests/report/main.go "$@";
 
 if [ -z "${NO_OPEN:-}" ]; then
   if [ -s "$REPORT_OUTPUT" ]; then
-    open "$REPORT_OUTPUT"
+    open "$REPORT_OUTPUT" 2>/dev/null || \
```
Comment (nhooyr) [inline ci/e2ereport.sh:12]: Use `command -v` to check if they exist before opening them.

With `2>/dev/null` you won't get any error messages even if they're legit.

Comment (nhooyr) [reply ci/e2ereport.sh:12]: Grep other scripts for `command -v` to see how it's used.

**Review by nhooyr — CHANGES_REQUESTED** [review]

File: ci/e2ereport.sh (hunk 1 above)
Comment (exapsy) [reply ci/e2ereport.sh]: Addressed on e8906ef :D

**Comment by exapsy** [comment]
Closing to ensure signed commits from my side

**Pull request closed without merging**